impeller/entity/contents/content_context.h

```diff
@@ -279,11 +277,17 @@ struct ContentContextOptions {
   bool is_for_rrect_blur_clear = false;
 
   struct Hash {
-    constexpr std::size_t operator()(const ContentContextOptions& o) const {
-      return fml::HashCombine(
-          o.sample_count, o.blend_mode, o.stencil_compare, o.stencil_operation,
-          o.primitive_type, o.color_attachment_pixel_format,
-          o.has_stencil_attachment, o.wireframe, o.is_for_rrect_blur_clear);
+    constexpr uint64_t operator()(const ContentContextOptions& o) const {
+      return static_cast<uint64_t>(o.sample_count) << 56 |                   //
```

Comment:
nit: I think this would be a bit easier to read if you count up instead of down.

Reply:
Done